REQUIRE

```diff
@@ -1,4 +1,3 @@
-julia 0.7
-RecursiveArrayTools 0.17.0
-AbstractOperators 0.1.0
+julia 1.0
 ProximalOperators 0.8.0
+RecursiveArrayTools
```

Comment:
I think we could be potentially independent of `RecursiveArrayTools` here since `ArrayPartition <: AbstractArray`...

Reply:
Good catch, I intended to put this only as a test requirement.

I actually think that the `REQUIRE` files should now go? According to the discussion [here](https://discourse.julialang.org/t/switching-package-registration-systems-soon/22677) only `Project.toml` is now required